README.md

```diff
@@ -11,6 +11,9 @@ The "source of truth" for the event data format is in the form of
 See [proto/README.md](proto/README.md) for more
 information about the messages used to describe CloudEvents.
 
+See [examples/README.md](proto/README.md) for examples of
```

Comment:
```suggestion
See [examples/README.md](examples/README.md) for examples of
```

Reply:
I believe it would be a lot easier to maintain and update these samples if the contents were next to the protos, so I've done that.